extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -191,6 +192,12 @@ describe('amp-a4a', () => {
     expect(child.src).to.match(/^https?:[^?#]+nameframe(\.max)?\.html/);
     const nameData = child.getAttribute('name');
     expect(JSON.parse.bind(null, nameData), nameData).not.to.throw(Error);
+    const attributes = JSON.parse(nameData);
```

Comment:
This is parsing `nameData` twice.  Is it possible to parse it once?  Maybe something like
```javascript
let nameResult = null;
expect(nameResult = JSON.parse.bind(null, nameData), nameData).not.to.throw.(Error);
```
?

Also, at this point do we expect the creative content of the name attribute to be cleared?  Either way, should maybe put an expectation on the shape of this data structure here.